### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by xerta555**

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -167,12 +169,14 @@
...
+  
+    <string name="about_source" translatable="false">https://github.com/LSPosed/LSPosed/</string>
 
     <string name="app_destroyed">Cette application à été détruite, merci de vous assurer que vous l\'avez bien téléchargé depuis la source officielle.</string>
     <string name="outdated_manager">Version du Gestionnaire LSPosed et du noyau LSPosed ne sont pas cohérents. Merci de réinstaller la version correspondante.</string>
     <string name="lsposed_not_active">LSPosed n\'est pas installé ou n\'est pas actif.</string>
     <string name="verbose_log_not_avaliable">Journal détaillé indisponible, si vous l\'avez juste activé sans redémarrer, essayez à nouveau après avoir redémarré.</string>
     <string name="NotInstall">Non installé</string>
     <string name="NotInstallDetail">LSPosed n\'est pas installé</string>
+    <string name="translators"><![CDATA[<a href="https://github.com/LSPosed/LSPosed">LSPosed</a>]]></string>
```
Comment (tehcneko) [inline app/src/main/res/values-fr/strings.xml:181]: Add your name to here

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -167,12 +169,14 @@
     <string name="module_release_load_more">Afficher les anciennes versions</string>
     <string name="module_release_no_more">Pas d\'autres versions</string>
     <string name="repo_load_failed">Échec de chargement du dépôt des modules: %s</string>
-
+  
+    <string name="about_source" translatable="false">https://github.com/LSPosed/LSPosed/</string>
```
Comment (tehcneko) [inline app/src/main/res/values-fr/strings.xml:173]: Remove this since it's translatable="false

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -19,6 +19,7 @@
   -->
 
 <resources>
+    <string name="app_name" translatable="false">LSPosed</string>
```
Comment (tehcneko) [inline app/src/main/res/values-fr/strings.xml:22]: and this

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (xerta555) [reply app/src/main/res/values-fr/strings.xml:181]: I have to add my Nickname instead of the URL ??

Comment (tehcneko) [reply app/src/main/res/values-fr/strings.xml:181]: <![CDATA[Your name here]]>

Comment (tehcneko) [reply app/src/main/res/values-fr/strings.xml:181]: URL is ok

Comment (xerta555) [reply app/src/main/res/values-fr/strings.xml:181]: so:
`<string name="translators"><![CDATA[<a href="https://github.com/LSPosed/LSPosed">MY-NAME</a>]]></string>`
?

Comment (tehcneko) [reply app/src/main/res/values-fr/strings.xml:181]: change link to your website or just remove whole <a > tag and write

Comment (xerta555) [reply app/src/main/res/values-fr/strings.xml:181]: Ah ok ok

**Comment by xerta555** [comment]
Good now ?

**Review by yujincheng08 — APPROVED** [review]

File: app/src/main/res/values-fr/strings.xml [hunk 4]
```diff
@@ -168,11 +169,11 @@
...
     <string name="repo_load_failed">Échec de chargement du dépôt des modules: %s</string>
 
-
     <string name="app_destroyed">Cette application à été détruite, merci de vous assurer que vous l\'avez bien téléchargé depuis la source officielle.</string>
     <string name="outdated_manager">Version du Gestionnaire LSPosed et du noyau LSPosed ne sont pas cohérents. Merci de réinstaller la version correspondante.</string>
     <string name="lsposed_not_active">LSPosed n\'est pas installé ou n\'est pas actif.</string>
     <string name="verbose_log_not_avaliable">Journal détaillé indisponible, si vous l\'avez juste activé sans redémarrer, essayez à nouveau après avoir redémarré.</string>
     <string name="NotInstall">Non installé</string>
     <string name="NotInstallDetail">LSPosed n\'est pas installé</string>
+    <string name="translators"><![CDATA[<aRomXDA</a>]]></string>
```
Comment (yujincheng08) [inline app/src/main/res/values-fr/strings.xml:178]: No its wrong.

**Review by yujincheng08 — CHANGES_REQUESTED** [review]

**Review by tehcneko — APPROVED** [review]

**Pull request merged**